pom.xml

```diff
@@ -0,0 +1,177 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>com.uber</groupId>
+    <artifactId>kafkaSpraynozzle</artifactId>
+    <packaging>jar</packaging>
+    <version>0.1.0</version>
+    <name>Kafka Spraynozzle</name>
+    <dependencies>
+        <dependency>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
+            <version>3.8.1</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.kafka</groupId>
+            <artifactId>kafka</artifactId>
+            <version>0.7.2</version>
+        </dependency>
+        <dependency>
+        	<groupId>com.101tec</groupId>
+        	<artifactId>zkclient</artifactId>
+        	<version>0.5</version>
+        </dependency>
+        <dependency>
+        	<groupId>org.apache.httpcomponents</groupId>
+        	<artifactId>httpclient</artifactId>
+        	<version>4.3.6</version>
+        </dependency>
+        <dependency>
+        	<groupId>com.lexicalscope.jewelcli</groupId>
+        	<artifactId>jewelcli</artifactId>
+        	<version>0.8.7</version>
+        </dependency>
+        <dependency>
+        	<groupId>com.fasterxml.jackson.core</groupId>
+        	<artifactId>jackson-core</artifactId>
+        	<version>2.5.3</version>
+        </dependency>
+        <dependency>
+        	<groupId>com.fasterxml.jackson.core</groupId>
+        	<artifactId>jackson-databind</artifactId>
+        	<version>2.5.3</version>
+        </dependency>
+        <dependency>
+        	<groupId>com.fasterxml.jackson.core</groupId>
+        	<artifactId>jackson-annotations</artifactId>
+        	<version>2.5.3</version>
+        </dependency>
+        <dependency>
+        	<groupId>com.timgroup</groupId>
+        	<artifactId>java-statsd-client</artifactId>
+        	<version>3.1.0</version>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.curator</groupId>
```

Comment:
You sure you want to include this in the same change that brings in a different build tool?

These should probably be separate so we're sure there's no regression because the packages differ.